src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -39,56 +39,64 @@
     private static final Supplier<Double> leftCorrection = corrections.addConstantDouble("left correction", 1);
```

Comment:
ok you changed so much in this class I'm not going to even comment on everything,
just change all of these to the right things in `autonomous-testing`